src/config.rs

```diff
@@ -0,0 +1,135 @@
+use anyhow::{anyhow, Result};
+use clap::{App, Arg};
+
+use crate::runtime;
+
+pub const VERSION: &str = "0.4.0";
```

Comment:
nit: we can probably drop the type annotation here -- rustc will infer it for us, and it doesn't provide much value.

Reply:
Ah, right right.

Reply:
<img width="667" alt="Screen Shot 2022-10-04 at 11 04 28 AM" src="https://user-images.githubusercontent.com/54378333/193855369-0582c114-5908-4d4e-88d3-11ed84be9e1f.png">

Ah, just remembering. This is why I added the type annotation here. I was reading a bit about the _why_ behind this [here](https://users.rust-lang.org/t/why-is-type-declaration-necessary-in-constants/14200/3).